primo/opt_model/model_options.py

```diff
@@ -302,12 +401,13 @@ def solve_model(self, **kwargs):
             solver.set_gurobi_param("PoolSolutions", pool_size)
 
         # Solve the optimization problem
-        solver.solve(self._opt_model)
+        solver.solve(self._opt_model, tee=kwargs.pop("stream_output", True))
```

Comment:
Why are we using pop? What if a user wants to use the same kwargs dictionary in multiple calls?